libvmi/os/windows/core.c

```diff
@@ -50,6 +51,8 @@ win_ver_t ntbuild2version(uint16_t ntbuildnumber)
         case 9200:
         case 9600:
             return VMI_OS_WINDOWS_8;
+        case 18432:
```

Comment:
Are there any news concerning the weird NtBuildNumbers?
I really would like to see this merged (with all three numbers currently possible for Win10 ) 
> 10240 (TH1)
> 10586 (TH2)
> 14393 (RS1)

In my tests I did not see any other values than the expected ones, so returning VMI_FAILURE if no valid ntbuildnumber is found works well for me.